### PR TITLE
Remove dependency on volumes

### PR DESCRIPTION
**Pull request by mpuckett159**
Reality is I've been playing with Kubernetes and I wanted to try using it with this app, but I didn't want to deal with persistent volumes, so here we are.

This also makes the set up a bit easier just like in general so I guess win win. I checked though and it only costs an extra 3MB for the image so... what ever.

### CODE REVIEW

**Comment by mpuckett159** [comment]
Probably should have created this as a draft PR to wait on the other branch merge but w/e, this is complete now and ready for review.

File: db/Dockerfile [hunk 1]
```diff
@@ -5,4 +5,7 @@ COPY *.conf /data/
 COPY sql/* /docker-entrypoint-initdb.d/
 RUN chmod a+r /docker-entrypoint-initdb.d/*
 
+COPY seed/* /seed/
+RUN chmod a+r /seed/*
```
Comment (AetherUnbound) [inline db/Dockerfile:9]: Are these chmod steps necessary? It looks like the postgres container container gets run with the `root` user, so there shouldn't be any permissions issues :thinking:

Comment (mpuckett159) [reply db/Dockerfile:9]: Unsure I can try it out I guess. Doesn't really matter either way, though, I don't think.

Comment (mpuckett159) [reply db/Dockerfile]: You are correct, have removed.

Comment (AetherUnbound) [reply db/Dockerfile]: Pog

**Review by AetherUnbound — APPROVED** [review]

**Pull request merged**